### ISSUE TITLE
Need more maintainers

### ISSUE THREAD

**Issue by Rockstar04** [NewRelic-Python-Plugins/newrelic-python-agent#1]
I'm by no means a Python expert. To keep this project healthy and alive we need more people who can answer issues and review pull requests. If you want to help out please respond below and we can get you added to the org.

**Comment by wolph** [comment]
Sure, I have no problem doing my part :)

Btw, I would personally recommend recreating this repository not as a fork but as a new repository. Forks have several limitations which are a tad annoying. Especially when it comes to handling pull requests

**Comment by Rockstar04** [comment]
I agree that a fork isn't ideal, but I was hoping that the extra tie to the initial project would get more people to move over here. There is also the issue I opened #2, that talks about moving forward with new repositories, either as a monolithic agent that covers everything, or a more specialized set of python based agents.

**Comment by krzysztof-magosa** [comment]
I can help with RabbitMQ, Nginx, ElasticSearch, Memcached, PHP FPM and Redis.
The rest of agents is outside of my interest at the moment.

**Comment by tjstansell** [comment]
Glad to find that someone is reviving this project.  We're planning on using it to monitor RDS instances in AWS as a MySQL plugin.  I've already ported the official https://github.com/newrelic-platform/newrelic_mysql_java_plugin logic into a generic MySQL plugin for this and now I'm working on building a mechanism we can use to auto-detect changes to our RDS infrastructure and rebuild the config.  In doing so, I found that the helper library has an issue with config reloading, but I also just submitted a PR to fix that.  I also have a PR for this project that will fix a couple minor issues.
